src/QtLocationPlugin/QMLControl/OfflineMap.qml

```diff
@@ -368,6 +369,23 @@ Item {
                         visible:        _esriFact ? _esriFact.visible : false
                         font.pointSize: _adjustableFontPointSize
                     }
+
+                    Item { width: 1; height: 1; visible: _customURLFact ? _customURLFact.visible : false }
+                    QGCLabel { text: qsTr("Custom Map URL"); visible: _customURLFact ? _customURLFact.visible : false }
```

Comment:
`visible: _customURLFact && _customURLFact.visible`

Reply:
Sure. I did wonder about this pattern. This implementation was consistent with the rest of the file.